hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/BatchConsumer.java

```diff
@@ -60,22 +65,25 @@ public class BatchConsumer implements Consumer {
     private final BatchConsumerMetrics metrics;
     private MessageBatchReceiver receiver;
 
+    private final Map<SubscriptionPartition, Long> pendingOffsets = new HashMap<>();
```

Comment:
maybe `maxPendingOffsets`? to communicate that we are storing max offset per partition